nifi-nar-bundles/nifi-splunk-bundle/nifi-splunk-processors/src/main/java/org/apache/nifi/processors/splunk/GetSplunk.java

```diff
@@ -105,6 +106,20 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .defaultValue("8089")
             .build();
+    public static final PropertyDescriptor CONNECT_TIMEOUT = new PropertyDescriptor.Builder()
+            .name("Connection Timeout")
+            .description("Max wait time for connection to the Splunk server.")
+            .required(true)
```

Comment:
They should not be required and since we have a default they definitely don't need to be provided.